EmailValidator/Validation/EmailValidation.php

```diff
@@ -21,6 +21,8 @@ public function isValid($email, EmailLexer $emailLexer);
     /**
      * Returns the validation error.
      *
+     * Be careful about no-error does not necessarily mean email was valid.
```

Comment:
Well, for any given EmailValidation implementation, having no errors means it is valid in terms of that validator. What could be a case for an invalid email without error?